cocos2dx/base_nodes/CCAtlasNode.h

```diff
@@ -102,15 +102,15 @@ class CC_DLL CCAtlasNode : public CCNode, public CCRGBAProtocol, public CCTextur
     */
     virtual void updateAtlasValues();
 
-    virtual void draw(void);
+    virtual void draw(void) override;
```

Comment:
I started removing them, but then I felt like it makes it more obvious. It's also more visible than the `override` keyword, which sometimes gets pushed over the edge of the window/monitor.